app/views/admin/bulk_republishing/confirm_by_type.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :page_title, "Republish #{@bulk_content_type_metadata[:name]} '#{@content_type}'" %>
+<% content_for :title, "Are you sure you want to republish #{@bulk_content_type_metadata[:name]} '#{@content_type}'?" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @republishing_event)) %>
+
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      This will schedule <%= @bulk_content_type_metadata[:name] %> '<%= @content_type %>' to be republished.
```

Comment:
We've tended to use `schedule` elsewhere to represent things being added to queues, unless I've misunderstood e.g in `app/views/admin/bulk_republishing/confirm.html.erb` and `app/views/admin/republishing/index.html.erb`.

How strongly do you feel about this one?

Reply:
Yeah we discussed this a while back and made a card to fix the old instances of "schedule": https://trello.com/c/IrapH5gq/1177-replace-instances-of-scheduled-with-queued-in-the-republishing-interface

It's because we're just adding it to a queue to be performed when ready, rather than scheduling it for a specific time (using e.g. `perform_at` or `perform_in`)

Reply:
Ah yeah, this feature has really been going on a while, that conversation had slipped my mind...